vote/templates/vote/political_parties_votes.html

```diff
@@ -15,40 +15,65 @@ <h6 class='text-center'>Voting Date: 10/12/2021</h6>
         <thead>
           <tr>
             <th scope="col">No.</th>
-            <th scope="col">Political Parties</th>
-            {%for i in reg%}
+            <th scope="col">Regions</th>
+            {%for i in data%}
             <th scope="col">{{i}}</th>
             {%endfor%}
+            <th scope="col">REJECTED BALLOTS</th>
             <th scope="col">TOTAL</th>
             <th scope="col">Percentage (%)</th>
           </tr>
         </thead>
         <tbody>
-          {%for i in data%}
+          {%for i in final%}
             <tr>
-              <th scope="row">{{i.id}}</th>
-              <td>{{i}}</td>
-              <td></td>
-              
-              
+              <th scope="row">{{i.index}}</th>
```

Comment:
parties part, must be dynamic. We don't want to touch the source code once the project is done.

Reply:
Unfortunately with the kind of isolated projects , this would have to be the ideal solution.I only resloved the issues asbest as were described in the deliverable txt.You can take a look at a more optimal way and let me know your thoughts on that.Otherwise it will recquire me spending more time , which results to our terms of settlement agreement changing.